content/en/docs/project-user-guide/application-workloads/horizontal-pod-autoscaling.md

```diff
@@ -0,0 +1,139 @@
+
+3. In the **Create Service** dialog box, click **Stateless Service**.
+
+   ![stateless-service](/images/docs/project-user-guide/application-workloads/horizontal-pod-autoscaling/stateless-service.png)
+
+4. Set the Service name (for example, `hpa`) and click **Next**.
+
+   ![service-name](/images/docs/project-user-guide/application-workloads/horizontal-pod-autoscaling/service-name.png)
+
+5. Click **Add Container Image**, set **Image** to `mirrorgooglecontainers/hpa-example` and click **Use Default Ports**.
+
+   ![add-container-image](/images/docs/project-user-guide/application-workloads/horizontal-pod-autoscaling/add-container-image.png)
+
+6. Set the CPU request (for example, 0.15 cores) for each container, click **√**, and click **Next**.
+
+   {{< notice note >}}
+
+   * To use HPA based on CPU usage, you must set the CPU request for each container, which is the minimum CPU resource reserved for each container (for details, see the [official Kubernetes document](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)). The HPA feature compares the average Pod CPU usage with a target percentage of the average Pod CPU request.
+   * For HPA based on memory usage, you do not need to configure the memory request.
+
+   {{</ notice >}}
+
+   ![cpu-request](/images/docs/project-user-guide/application-workloads/horizontal-pod-autoscaling/cpu-request.png)
+
+7. Click **Next** on the **Mount Volumes** tab and click **Create** on the **Advanced Settings** tab.
+
+## Configure HPA
+
+1. Choose **Deployments** in **Workloads** on the left navigation bar and click the HPA Deployment (for example, hpa-v1) on the right.
+
+   ![hpa-deployment](/images/docs/project-user-guide/application-workloads/horizontal-pod-autoscaling/hpa-deployment.png)
+
+2. Click **More** and choose **Horizontal Pod Autoscaling** from the drop-down list.
+
+   ![horizontal-pod-autoscaling](/images/docs/project-user-guide/application-workloads/horizontal-pod-autoscaling/horizontal-pod-autoscaling.png)
+
+3. In the **Horizontal Pod Autoscaling** dialog box, configure the HPA parameters and click **OK**.
+
+   * **CPU Target Utilization**: Target percentage of the average Pod CPU request.
+   * **Memory Target Usage**: Target average Pod memory usage in MiB.
+   * **Min Replicas Number**: Minimum number of Pods.
+   * **Max Replicas Number**: Maximum number of Pods.
+
+   In this example, **CPU Target Utilization** is set to `60`, **Min Replicas Number** is set to `1`, and **Max Replicas Number** is set to `10`.
+
+   {{< notice note >}}
+
+   Ensure that the cluster can provide sufficient resource for all Pods when the number of Pods reaches the maximum. Otherwise, the creation of some Pods will fail.
+
+   {{</ notice >}}
+
+   ![hpa-parameters](/images/docs/project-user-guide/application-workloads/horizontal-pod-autoscaling/hpa-parameters.png)
+
+## Verify HPA
+
+This section verifies whether HPA functions properly.
```

Comment:
This section verifies whether HPA functions properly.
->
Create a Deployment thats send requests to the Service above to verify that HAP works as the CPU load increases.